src/main/java/ua/vedroid/cinema/model/User.java

```diff
@@ -16,6 +18,8 @@
     @Column(unique = true)
     private String email;
     private String password;
+    @ManyToMany
+    private List<Role> roles;
```

Comment:
```suggestion
    private Set<Role> roles;
```
set looks more logical - roles have unique constraint